### PR TITLE
Update all the Gems, all tests pass, everything seems to work.

### PR DESCRIPTION
**Pull request by minigeek**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -38,6 +38,8 @@ group :development do
 
   # passenger tasks in capistrano
   gem 'capistrano-passenger'
+  
+  gem 'meta_request'
```
Comment (minigeek) [inline Gemfile]: Oh shit, didn't mean to leave this in, you guys cool with it? It's a chrome extension to debug [RailsPanel](https://github.com/dejan/rails_panel).

Comment (pkoenig10) [inline Gemfile]: Fine with me. It can't hurt.

**Comment by pkoenig10** [comment]
Is this PR intended to address #121?

**Comment by minigeek** [comment]
Forgot to update other gems

**Comment by minigeek** [comment]
Also yes

**Pull request closed without merging**